pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -31,6 +31,7 @@
     PeriodIndex,
     Series,
     Timestamp,
+    array,
```

Comment:
i thought we are using pd.array rather than bare array (i could be out of date here)

Reply:
Did not know that. Changed it